news/10972.doc.rst

```diff
@@ -0,0 +1,2 @@
+The wheel command does now reference the build interface section instead od stating the legacy
```

Comment:
```suggestion
The wheel command does now reference the build interface section instead of stating the legacy
```
typo ^^

Reply:
Also, "does now reference" should be "now references". I don't know how to do a suggested change to a suggested change, sorry ;-)

Reply:
I think it should be like this, according to @pfmoore and @ichard26:

```suggestion
The wheel command now references the build interface section instead of stating the legacy
```